### PR TITLE
Modal size is flickering

### PR DESCRIPTION
**Pull request by exentrich**
When modal appears is not fullscreen, only after a moment. Looks like a bug.
So i suggest to update style to left:0, top:0, right: 0, bottom: 0

### CODE REVIEW

**Comment by brentvatne** [comment]
Please format the code in the same way as the rest of the codebase :) Each of the style properties should be on their own line.

File: Libraries/Modal/Modal.js [hunk 1]
```diff
@@ -65,11 +65,11 @@ Modal.propTypes = {
 var styles = StyleSheet.create({
   modal: {
     position: 'absolute',
+    left:0,top:0,right:0,bottom:0
```
Comment (javache) [inline Libraries/Modal/Modal.js]: This doesn't seem required. The element with style `modal` shouldn't be taking up room in the layout tree.

**Comment by mbrgm** [comment]
I can confirm the described behaviour. If the modal is created without animation and children of the modal are positioned with flexbox, there is a short flicker upon display.

**Comment by fangmobile** [comment]
confirmed. you can see the flicker even in UIExplorer Modal example. Do need to turn off animation. 
When animation is on there is anther issue when a line moves up on the screen. That might be  another issue.

**Comment by facebook-github-bot-0** [comment]
@javache would you mind taking a look at this pull request? It's been a while since the last commit was reviewed.

**Comment by mkonicek** [comment]
@exentrich Can you rebase please? Once you rebase we can review.

**Comment by dmmiller** [comment]
@mkonicek This will break some internal code as I discovered recently when I made a similar change while open sourcing the Android implementation.

**Comment by facebook-github-bot-1** [comment]
It's been a while since the last commit was reviewed and the labels show this pull request needs review. Based on the blame information for the files in this pull request we identified @jemise111 as a potential reviewer. Could you take a look please or cc someone with more context?

**Comment by javache** [comment]
With https://github.com/facebook/react-native/commit/2b69ec25897565ebd31f3430daa18e21ec957f9d this should no longer be relevant.

**Pull request closed without merging**